src/create-mock.ts

```diff
@@ -1,3 +1,3 @@
 import { DeepPartial } from './partial/deepPartial';
 
-export declare function createMock<T extends object>(values?: DeepPartial<T>): T;
+export declare function createMock<T extends object | undefined>(values?: DeepPartial<T>): T;
```

Comment:
If you check my comment below I wouldn't set this as undefined unless there is a specific reason

Reply:
This was solely intended for the type query test covering `createMock<typeof undefined>`, there, `object` isn't enough. The meaning of it, I'm not sure 😛

Reply:
I would prefer to remove the eslint in the test. The usual behavior is always with an object.